### ISSUE TITLE
Thank you for the great work.

### ISSUE THREAD

**Issue by bit-lang** [nbarkhina/DosWasmX#14]
One little problem is, if I try to select a cpu higher than 486, the page will freeze.

**Comment by bit-lang** [comment]
Using 486 as cpu will still cause the page to freeze at some stage. falling back to auto works.

**Comment by nbarkhina** [comment]
yes the "auto" and "max" options are the best to use. Some of the higher cycle CPU's like 50000 or above require very powerful computers to run. I included those just to future proof it.

**Comment by MajesticWafer** [comment]
> yes the "auto" and "max" options are the best to use. Some of the higher cycle CPU's like 50000 or above require very powerful computers to run. I included those just to future proof it.

Would an Intel Core i7 do the trick?

**Comment by bit-lang** [comment]
> > yes the "auto" and "max" options are the best to use. Some of the higher cycle CPU's like 50000 or above require very powerful computers to run. I included those just to future proof it.
> 
> Would an Intel Core i7 do the trick?

I'm using 13-gen core i5.
hope someone can test using an i7
